packages/marketing-analytics-browser/README.md

```diff
@@ -31,11 +31,6 @@ Alternatively, the package is also distributed through a CDN. Copy and paste the
 
 <!-- README_SNIPPET_BLOCK -->
 ```html
-<script type="text/javascript">
```

Comment:
Maybe remove the whole block, if we want to remove the snippet here.

Reply:
ah, this is not intentional. will remove this diff

Reply:
reverted the change